JavaAssignment/src/ravi/ArrayEqualityCheck.java

```diff
@@ -0,0 +1,42 @@
+package ravi;
+import java.util.Scanner;
+
+/*Class can be used to check if two given arrays are equal*/
+public class ArrayEqualityCheck {
+
+	int[] takeValueFromUserForFirstArray() { // Method will take first set of array as input
+		Scanner scan = new Scanner(System.in);
```

Comment:
If some else want to use with available param how would they use it
for user want to pass data from excel sheet.
method should take param from only main method